var/spack/repos/builtin/packages/psp-validation/package.py

```diff
@@ -27,7 +28,7 @@ class PspValidation(PythonPackage):
     depends_on('py-numpy@1.10:', type='run')
     depends_on('py-tqdm@4.0:', type='run')
     depends_on('py-bglibpy@4.0:', type='run')
-    depends_on('py-bluepy@0.14.1:', type='run')
+    depends_on('py-bluepy@0.14.3:', type='run')
     depends_on('py-efel@3.0.39:', type='run')
```

Comment:
That EFEL version does not exist!

https://github.com/BlueBrain/spack/blame/develop/var/spack/repos/builtin/packages/py-efel/package.py#L40

Reply:
Thanks, somehow it was still building when I tried on BB5...

Reply:
It may fail silently since this is just a run dependency… I'll have to look into it if we can change that behavior